test/e2e/installplan_e2e_test.go

```diff
@@ -2715,6 +2715,273 @@ var _ = Describe("Install Plan", func() {
 			Expect(ref.Kind).To(Equal("ConfigMap"))
 		}
 	})
+	It("uses the OLM client for CRD installation", func() {
+		c := newKubeClient()
+		crc := newCRClient()
+
+		By("using a scoped serviceaccount specifified in the operatorgroup")
+		ns, err := c.KubernetesInterface().CoreV1().Namespaces().Create(context.TODO(), &corev1.Namespace{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: genName("ns-"),
+			},
+		}, metav1.CreateOptions{})
+		Expect(err).ToNot(HaveOccurred())
+		defer c.KubernetesInterface().CoreV1().Namespaces().Delete(context.TODO(), ns.GetName(), metav1.DeleteOptions{})
+
+		// create SA
+		sa := &corev1.ServiceAccount{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      genName("sa-"),
+				Namespace: ns.GetName(),
+			},
+		}
+		_, err = c.KubernetesInterface().CoreV1().ServiceAccounts(ns.GetName()).Create(context.TODO(), sa, metav1.CreateOptions{})
+		Expect(err).ToNot(HaveOccurred())
+
+		// create role that includes permission for installing the kiali operator, but not CRDs
+		// see https://github.com/operator-framework/operator-lifecycle-manager/blob/master/doc/design/scoped-operator-install.md
+		role := &rbacv1.ClusterRole{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: genName("role-"),
+			},
+			Rules: []rbacv1.PolicyRule{
+				{
+					APIGroups: []string{"operators.coreos.com"},
+					Resources: []string{"subscriptions", "clusterserviceversions"},
```

Comment:
This brings up another good question: shouldn't resolvable OLM kinds _always_ be applied with OLM's privileges? 

Requiring explicit permissions for OLM resources (of the OperatorGroup ServiceAccount) seems like _YetAnotherHoop_ ™️

Reply:
I totally agree. Why does an operator's SA need knowledge of OLM's APIs to be installed by OLM is unclear -- it would make more sense if this was implied, as you said by using the OLM client for these types. 

I got these from https://docs.openshift.com/container-platform/4.5/operators/admin/olm-creating-policy.html#olm-policy-fine-grained-permissions_olm-creating-policy 

Seems like using your own SA in the OG means you must acknowledge the OLM types - since this is more advanced behavior I can see why this makes sense as is, but yeah changing it could make sense